index.php

```diff
@@ -20,15 +20,23 @@
 
     <h2>NEW BOOK?</h2>
 
-    <form>
+    <form method="post">
         <label>Title: <input type="text" name="title" placeholder="enter text here" /></label><br>
         <label>Author: <input type="text" name="author" placeholder="enter text here" /></label><br>
-        <label>Year Released: <input type="text" name="released" placeholder="enter number here" /></label><br>
+        <label>Year Released: <input type="number" name="released" placeholder="enter number here" /></label><br>
         <label>Genre: <input type="text" name="genre" placeholder="enter text here" /></label><br>
-        <label>Page Count: <input type="text" name="page_count" placeholder="enter number here" /></label><br>
+        <label>Page Count: <input type="number" name="page_count" placeholder="enter number here" /></label><br>
         <input type="submit" value="Add to Collection"/>
     </form>
 
+    <?php
+
+    if( (!empty($_POST['title'])) && (!empty($_POST['author'])) && (!empty($_POST['released'])) && (!empty($_POST['genre'])) && (!empty($_POST['page_count'])) ) {
+        insertToDB($db, $_POST['title'], $_POST['author'], $_POST['genre'], $_POST['page_count'], $_POST['released']);
+    }
+
+    ?>
+
     <h2>BOOK COLLECTION</h2>
```

Comment:
Are you going to do an error message if they are empty?

Reply:
Good shout, totally let it slip my mind